### DISCUSSION TITLE
More 6000 km with Cruiser in France Spain and Portugal

### DISCUSSION THREAD

**Discussion by jclaudej** [devemux86/cruiser#182] [Q&A] [answered]
First of all, thank you very much for this fantastic application. I use Cruiser with Wunderlinq on my R1250RT and it's very nice to be able to use Zoom + or - while riding without letting go of the handlebars.
Today I have a problem with Brouter because I find that if I want to create a route and my position is not next to a road, it is impossible to create my route. I need to create a waypoint on the next route so I can finally create my route.
It's strange because with other GPS apps I haven't encountered this problem. Brouter is not easy to learn and will probably take me a lot of time to manage it ;)

**Comment by devemux86** [comment]
@jclaudej thanks for using Cruiser and congratulations on going such a long distance with it!

If you have any questions or ideas for new features and improvements, please let me know.

> I find that if I want to create a route and my position is not next to a road, it is impossible to create my route.

Do you get the message "BRouter: viaX-position not mapped in existing datafile"?

You can increase the "Settings | Routing | BRouter | Waypoints search distance" to be more than 250 m.
Does it help?

Please also see this discussion:
- https://github.com/devemux86/cruiser/discussions/170

<img src="https://github.com/devemux86/cruiser/assets/3484020/68bc8997-01da-44ba-a216-fb3ad2d2a7df" width="50%"/>

**Comment by jclaudej** [comment]
Thank a lot for your advice but when I wanted created an itinerary I was hotel santuario de sancho abarca in Spain (Aragon) and if you look for an itinerary for this destination only Google Map, Waze and Go ride found an itinerary perhaps because this road was be built 2 years ago and in the past the distance between this hotel and the next road was about 17km. Perhaps Brouter can’t manage this distance without waypoint additional 😉

![image](https://github.com/devemux86/cruiser/assets/126952624/119bce72-e0f5-44c0-b723-80ef25cfae92)

**Reply by devemux86** [reply]
Can you post a route example as file or url to better see what you describe?

Or do you mean the search for the hotel name in Cruiser's search screen?

**Reply by devemux86** [reply]
@jclaudej thanks, now I see the image.

This means that the end waypoint cannot be set on the road network by BRouter.
Do you use BRouter (offline)?

Have you increased the "Waypoints search range" mentioned [above](https://github.com/devemux86/cruiser/discussions/182#discussioncomment-6019092)?

**Comment by devemux86** [comment]
@jclaudej 

The reason is that this way is `highway=track tracktype=grade1`: https://www.openstreetmap.org/way/27077063

Next Cruiser will have an option to use such road tracks and then the BRouter routing works.
More details are explained here: https://github.com/devemux86/cruiser/discussions/180

You can already route this road using a [modified BRouter profile](https://github.com/devemux86/cruiser/discussions/180#discussioncomment-6011300) or better wait for the next Cruiser release!

<img src="https://github.com/devemux86/cruiser/assets/3484020/9c60b224-02a9-4fca-82b6-484a040edebf" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/acb41934-3bd3-4e93-8864-eb7ec86ab600" width="50%"/>

**Comment by jclaudej** [comment]
look forward to the next version of Cruiser 😉✌️

**Comment by devemux86** [comment, marked as answer]
BRouter routing on `highway=track` should work in [Cruiser 3.0.21](https://play.google.com/store/apps/details?id=gr.talent.cruiser) with its `car-cruiser` profile for Car / Motorcycle.

Activate the "Settings | Routing | BRouter (offline) | Car, Motorcycle | use_tracks" setting.

**Reply by jclaudej** [reply]
Thank I installed this last version and I will test soon ✌️😉
